pages/admin/request/create-replacement.tsx

```diff
@@ -58,19 +72,120 @@ export default function CreateReplacement() {
+        router.push(`/admin/request/${data.createReplacementApplication.applicationId}`);
+      }
+    },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+      });
+    },
+  });
+
+  const handleSubmit = async (event: SyntheticEvent) => {
+    event.preventDefault();
+    if (applicantId == undefined) {
+      toast({
+        status: 'error',
+        description: 'You must select a permit holder for a Replacement Request.',
+      });
+      return;
+    }
+    await submitReplacementApplication({
+      variables: {
+        input: {
+          applicantId,
+          firstName: permitHolderInformation.firstName,
+          lastName: permitHolderInformation.lastName,
+          phone: permitHolderInformation.phone,
+          email: permitHolderInformation.email,
+          addressLine1: permitHolderInformation.addressLine1,
+          addressLine2: permitHolderInformation.addressLine2,
+          city: permitHolderInformation.city,
+          postalCode: permitHolderInformation.postalCode,
+          reason: reasonDetails.reason,
+          lostTimestamp: reasonDetails.lostTimestamp,
+          lostLocation: reasonDetails.lostLocation,
+          description: reasonDetails.description,
+          paymentMethod: paymentDetails.paymentMethod,
+          donationAmount: paymentDetails.donationAmount,
+          shippingAddressSameAsHomeAddress: paymentDetails.shippingAddressSameAsHomeAddress,
+          shippingFullName: paymentDetails.shippingFullName,
+          shippingAddressLine1: paymentDetails.shippingAddressLine1,
+          shippingAddressLine2: paymentDetails.shippingAddressLine2,
+          shippingCity: paymentDetails.shippingCity,
+          shippingProvince: paymentDetails.shippingProvince,
+          shippingPostalCode: paymentDetails.shippingPostalCode,
+          billingAddressSameAsHomeAddress: paymentDetails.billingAddressSameAsHomeAddress,
+          billingFullName: paymentDetails.billingFullName,
+          billingAddressLine1: paymentDetails.billingAddressLine1,
+          billingAddressLine2: paymentDetails.billingAddressLine2,
+          billingCity: paymentDetails.billingCity,
+          billingProvince: paymentDetails.billingProvince,
+          billingPostalCode: paymentDetails.billingPostalCode,
+        },
+      },
+    });
+  };
+
   return (
     <Layout>
       <GridItem display="flex" flexDirection="column" colSpan={12} paddingX="108px">
         <Flex>
           <Text textStyle="display-large">
-            {`New Replacement Request (User ID:`}
+            {applicantId != undefined
+              ? `New Replacement Request (User ID:`
+              : `New Replacement Request`}
             <Box as="span" color="primary">
               {' '}
               <Link href={`/admin/request/${permitHolderID}`}>
                 <a>{permitHolderID}</a>
               </Link>
             </Box>
-            )
+            {applicantId != undefined && ')'}
```

Comment:
We can write this block with one `applicantId !== undefined` condition. Wrap the user ID with brackets in a React fragment (`<></>`)